python/smashbox/utilities/__init__.py

```diff
@@ -37,17 +37,19 @@ def reset_owncloud_account(reset_procedure=None, num_test_users=None):
         logger.info('reset_owncloud_account (%s) for %d users', reset_procedure, num_test_users)
 
     if reset_procedure == 'delete':
-        delete_owncloud_account(config.oc_account_name)
-        create_owncloud_account(config.oc_account_name, config.oc_account_password)
+        #delete_owncloud_account(config.oc_account_name)
+        #create_owncloud_account(config.oc_account_name, config.oc_account_password)
         login_owncloud_account(config.oc_account_name, config.oc_account_password)
-
-        if num_test_users is not None:
```

Comment:
It could be left like it was, just commenting in smashbox conf that only for the sharing mode.